Google.Api.Gax.Grpc/Rest/RestMethod.cs

```diff
@@ -143,13 +144,15 @@ internal HttpRequestMessage CreateRequest(IMessage protoRequest, string host)
             var uri = host is null ? new Uri(uriPathWithParams, UriKind.Relative) : new UriBuilder { Host = host, Path = uriPathWithParams }.Uri;
             
             var content = transcodingResult.Body is null ? null : new StringContent(transcodingResult.Body, Encoding.UTF8, s_applicationJsonMediaType);
-            
-            return new HttpRequestMessage
+
+            var httpRequestMessage = new HttpRequestMessage
```

Comment:
Revert all the changes to this file now?

Reply:
:+1: